app/src/main/java/com/sdp13epfl2021/projmag/database/impl/cache/OfflineCachedUserdataDatabase.kt

```diff
@@ -55,23 +57,40 @@ class OfflineCachedUserdataDatabase(
 
     private fun loadUsersData() {
         try {
-            usersDir.listFiles()?.forEach { file ->
-                when (file.name) {
-                    cvFilename -> loadFromFile(file, CurriculumVitae::class)
-                        ?.let { cv ->
-                            file.parent?.let { userID ->
-                                cvs.put(userID, cv)
+            usersDir.listFiles()?.forEach { userDir ->
+                userDir.listFiles()?.forEach { file ->
```

Comment:
It took me some time to figure out the difference between `usersDir` and `userDir`, it would be good to rename to avoid mistakes and improve readability

Reply:
I changed the usersDir to usersRootDir